### DISCUSSION TITLE
Speedtest

### DISCUSSION THREAD

**Discussion by Jeiel0rbit** [termux/termux-packages#17520] [Ideas]
Know more: https://www.speedtest.net/apps/cli

Add the package in Termux speedtest. It is possible to test internet latency via terminal. A reasonably useful resource.

`pkg install speedtest`

**Comment by licy183** [comment]
It is not a open source software, and it is not compiled for Termux, so I don't think it should be packaged.

**Reply by agnostic-apollo** [reply]
If its not open source, then no need to think more, since it doesn't comply with our packaging policy and you can close.

https://www.reddit.com/r/HomeNetworking/comments/dpalqu/speedtestnet_just_launched_an_official_c_cli/f5tmibx
